contrib/iso2usb.py

```diff
@@ -29,43 +29,42 @@
 import sys
 import stat
 import getopt
+import subprocess
 
-import executil
-
-def fatal(e):
-    print >> sys.stderr, 'Error: ' + str(e)
+def fatal(e: Any) -> NoReturn:
+    print('Error: ' + str(e), file=sys.stderr)
     sys.exit(1)
 
-def usage(e=None):
+def usage(e: Any=None) -> NoReturn:
```

Comment:
I'm not a fan of this style personally. I think 0 spaces is too dense, but spaces around equals make it too sparse and more difficult to pick out individual arguments. While `:` is a thin character and benefits from nearby whitespace, I argue that `=` is already an obvious deliminator. (Either-way this is clearly something that'll need to be decided on one way or another in our style guide)

Reply:
I don't feel strongly enough about this to take a firm stand.

However, considering this a little more, my only concern around no spaces around the `=` is that at a glance, when looking at `e: Any=None`, it appears (to me at least) to be a variable named `Any` with a value of `None`, rather than a variable `e`. With the extra spaces, it's not at all clear from a glance, but at least I don't make a wrong assumption.

When using specific types (from the `typing` library) it's not too bad as they are classes, so it's fairly clear what is going on. But there are times when a specific type might be used (e.g. `str`). Having said that, in the specific case of `str`, it's probably better to default to the one you used somewhere (I can't find it now and don't recall exactly what it was; bu something like `StringLike`?!).